File: app/src/main/java/com/mmtran/turtlesoccer/models/ConfederationsViewModel.java
package com.mmtran.turtlesoccer.models;

import androidx.lifecycle.LiveData;
import androidx.lifecycle.MutableLiveData;
import androidx.lifecycle.ViewModel;

public class ConfederationsViewModel extends ViewModel {

    private MutableLiveData<String> mText;

    public ConfederationsViewModel() {
        mText = new MutableLiveData<>();
        mText.setValue("FIFA and Regional Confederations");
    }

    public LiveData<String> getText() {
        return mText;
    }
}

File: app/src/main/java/com/mmtran/turtlesoccer/models/ConfederationsViewModel.kt
package com.mmtran.turtlesoccer.models

import androidx.lifecycle.LiveData
import androidx.lifecycle.MutableLiveData
import androidx.lifecycle.ViewModel

class ConfederationsViewModel : ViewModel() {

    private val _confederationList: MutableLiveData<List<Confederation>> = MutableLiveData()

    val confederationList: LiveData<List<Confederation>>
        get() = _confederationList

    fun setConfederationList(confederationList: List<Confederation>) {
        _confederationList.postValue(confederationList)
    }
}
